### DISCUSSION TITLE
Knobby on ESP32 S3?

### DISCUSSION THREAD

**Discussion by Linkyo75** [quadule/knobby#11] [Q&A] [answered]
Hi,
I ordered a ESP32 S3 as it was difficult to find the same version you're using in your project. I found out however that I would not be able to flash the ESP. May I ask where I should look at to make changes to the files and be able to use the firmware on the ESP32 S3?
Happy naturally to share the updated files once I've succeeded (assuming I will, may take me a while) so that the community can enjoy it.
Many thanks in advance.
Linkyo

**Comment by quadule** [comment, marked as answer]
Hi @Linkyo75!

Are you talking about the T-Display S3? I had one already so I just added a new release to support it.

Let me know if it works for you. I'd love to see a photo of your completed build too!

—Milo

**Reply by Linkyo75** [reply]
Hi Milo @quadule! 

Thanks a lot for that, this is very much appreciated. I can also share my 3D design for the case (not as skilled as you are though). 
Turns out that I bought the Diymore ESP32 S3 W with 0.96 inch OLED Display, and somehow this new version does not work with it either. May be due to display pins not being the same as for your version. I will check the code in more details and see if I can figure it out, but in the meantime, just decided to buy the T-Display S3 from AliExpress. Let's see in a few weeks if that works.
Thanks again for your reactivity, highly appreciated. 

Linkyo

**Reply by quadule** [reply]
Ah yeah, that sounds like a different display but I'm having trouble finding info about that model.

I had been meaning to try out the T-Display S3 for a while, so thanks for the push! I originally bought it for a [homeThing](https://github.com/landonr/homeThing) build, but porting Knobby turned out to be easier than expected. It isn't really making the most of the extra screen area yet though — planning to try a vertical UI orientation next.
